extension/src/popup/App.tsx

```diff
@@ -17,6 +17,7 @@ import { reducer as settings } from "popup/ducks/settings";
 import { Router } from "./Router";
 
 import "./styles/global.scss";
+import "./styles/fullScreen.scss";
```

Comment:
💭 thoughts on importing in the FullscreenStyle.ts file? (I _think_ that would work) To keep the style coupled with the component

Reply:
Good call - that's better!